packages/transaction-controller/CHANGELOG.md

```diff
@@ -7,6 +7,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [36.2.0]
+
+### Changed
+
+- Bump `@metamask/controller-utils` from `^11.2.0` to `^11.3.0` ([#4688](https://github.com/MetaMask/core/pull/4688))
+
+### Removed
+
+- Removed trace types from `TransactionController`, instead import them from `@metamask/controller-utils` ([#4655](https://github.com/MetaMask/core/pull/4655))
```

Comment:
Should we highlight this as a breaking change?

Reply:
We might also want to enumerate the affected type names:

```suggestion
- **BREAKING:** Removed types `TraceContext`, `TraceRequest`, `TraceCallback` from `TransactionController` ([#4655](https://github.com/MetaMask/core/pull/4655))
  - These types have been moved to `@metamask/controller-utils`.
```

Reply:
Good point but this is not a breaking change, those types never exported from `TransactionController`, just used internally.

Reply:
Ah I see that they're not listed in `index.ts`. In that case, we could also remove this entry altogether, although leaving it also works if its relevant to the team.